docs/aws/audit/iamcompliance/rules/account_with_multiple_admins.mdx

```diff
@@ -23,6 +23,233 @@ CBP
+
+3. For each user, under the "Permissions" tab, check the attached policies. If a user has "AdministratorAccess" or any other policy that grants full access to AWS services and resources, they are considered an admin.
+
+4. Count the number of users with admin access. If the number is high (based on your organization's policy), then it's a misconfiguration. The exact number that constitutes "too many" can vary depending on the size and nature of your organization.
+</Accordion>
+
+<Accordion title='Using CLI'>
+1. Install and configure AWS CLI: Before you can start using AWS CLI, you need to install it on your local machine. You can download it from the official AWS website. After installation, you need to configure it with your AWS account credentials. You can do this by running the command `aws configure` and then entering your AWS Access Key ID, Secret Access Key, Default region name, and Default output format when prompted.
+
+2. List all IAM users: Use the AWS CLI command `aws iam list-users` to list all the IAM users in your AWS account. This command will return a JSON object that contains information about all IAM users.
+
+3. List all IAM policies: Use the AWS CLI command `aws iam list-policies` to list all the IAM policies in your AWS account. This command will return a JSON object that contains information about all IAM policies.
+
+4. Check for admin policies: Now, you need to check if any of the listed IAM users have admin policies attached to them. You can do this by running the AWS CLI command `aws iam list-attached-user-policies --user-name <username>` for each user. Replace `<username>` with the name of the IAM user. This command will return a list of all policies attached to the specified IAM user. If the list contains an admin policy, then the user has admin privileges. Repeat this step for all IAM users. If the number of users with admin privileges is too high, then your AWS account has too many admins.
+</Accordion>
+
+<Accordion title='Using Python'>
+1. Install the necessary AWS SDK for Python (Boto3) if you haven't done so already. You can install it using pip:
+
+```python
+pip install boto3
+```
+
+2. Import the necessary modules and create a session using your AWS credentials:
+
+```python
+import boto3
+
+session = boto3.Session(
+    aws_access_key_id='YOUR_ACCESS_KEY',
+    aws_secret_access_key='YOUR_SECRET_KEY',
+    aws_session_token='SESSION_TOKEN',
+)
+```
+
+3. Use the IAM client to get a list of all IAM users:
+
+```python
+iam = session.client('iam')
+
+users = iam.list_users()['Users']
+```
+
+4. For each user, check if they have administrative privileges. This can be done by checking if the user has the 'AdministratorAccess' policy attached:
+
+```python
+admin_count = 0
+for user in users:
+    policies = iam.list_attached_user_policies(UserName=user['UserName'])['AttachedPolicies']
+    for policy in policies:
+        if policy['PolicyName'] == 'AdministratorAccess':
+            admin_count += 1
+
```

Comment:
<h2>Secret Detected: Secret Keyword</h2> <a href="https://console.cloudanix.com/code?t=secrets"> Click here for more details</a>